src/Microsoft.SqlTools.ServiceLayer/Security/UserData.cs

```diff
@@ -237,8 +238,13 @@ private void LoadUserData(CDataContainer context)
             if (SqlMgmtUtils.IsSql11OrLater(context.Server.ServerVersion))
             {
                 this.authenticationType = existingUser.AuthenticationType;
-                this.defaultLanguageAlias = LanguageUtils.GetLanguageAliasFromName(existingUser.Parent.Parent,
-                                                                existingUser.DefaultLanguage.Name);
+
+                if (context.Server.ServerType != DatabaseEngineType.SqlAzureDatabase)
+                {
+                    this.defaultLanguageAlias = LanguageUtils.GetLanguageAliasFromName(
+                                                                    existingUser.Parent.Parent,
+                                                                    existingUser.DefaultLanguage.Name);
```

Comment:
`existingUser.DefaultLanguage` is throwing an exception when connected to SQL DB.